Set_movable_separability_2/doc/Set_movable_separability_2/Concepts/CastingTraits_2.h

```diff
@@ -7,7 +7,9 @@
 
   \cgalRefines{DefaultConstructible,PolygonTraits_2}
 
-  \cgalHasModel Any CGAL kernel, e.g., CGAL::Exact_predicates_exact_constructions_kernel.
+  \cgalHasModelsBegin
+  \cgalHasModelsBare{Any CGAL kernel, e.g., CGAL::Exact_predicates_exact_constructions_kernel.}
```

Comment:
```suggestion
  \cgalHasModelsBare{Any \cgal kernel, e.g., CGAL::Exact_predicates_exact_constructions_kernel.}
```

Reply:
I'll correct this on.

I did a quick and dirty grep for `CGAL` in ci\ombination with `cgal` and got:
```
BGL/doc/BGL/PackageDescription.txt:584:\cgalPkgDescriptionBegin{CGAL and the Boost Graph Library,PkgBGL}
CGAL_ipelets/doc/CGAL_ipelets/PackageDescription.txt:4:\cgalPkgDescriptionBegin{CGAL Ipelets,PkgCGALIpelets}
GraphicsView/doc/GraphicsView/PackageDescription.txt:12:\cgalPkgDescriptionBegin{CGAL and the Qt Graphics View Framework,PkgGraphicsView}
Point_set_processing_3/include/CGAL/pointmatcher/compute_registration_transformation.h:507:       \cgalParamExtra{The logs generated by CGAL library does not get effected by this configuration.}
STL_Extension/doc/STL_Extension/PackageDescription.txt:15:\cgalPkgDescriptionBegin{STL Extensions for CGAL,PkgSTLExtension}
Shape_regularization/include/CGAL/Shape_regularization/regularize_contours.h:104:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_contours.h:227:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_segments.h:117:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_segments.h:355:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_segments.h:448:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_segments.h:541:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Shape_regularization/include/CGAL/Shape_regularization/regularize_segments.h:633:        \cgalParamDefault{a CGAL `Kernel` deduced from the point type,
Solver_interface/doc/Solver_interface/PackageDescription.txt:25:\cgalPkgDescriptionBegin{CGAL and Solvers, PkgSolverInterface}
```
maybe these should be looked at as well.